src/main/scala/fundamentals/level04/LogParser.scala

```diff
@@ -75,14 +77,9 @@ object LogParser {
     *
     * scala> getUnknowns(List(KnownLog(Info, 147, "mice in the air"), UnknownLog("blblbaaaaa")))
     * = List(UnknownLog("blblbaaaaa"))
-    **/
-  def getUnknowns(logs: List[LogMessage]): List[LogMessage] = ???
-
-  /**
-    * Define a function that returns the error log with the most recent (highest) timestamp.
     *
-    * What if we cannot find any error logs in the input?
-    */
+    * What if we get an empty line from the fileContent?
```

Comment:
This is good, but what you removed is valid too.

If there are no error logs, e.g. you have a List of Info logs, you want to return None.